### PR TITLE
Add dockerNet config option

### PR DESCRIPTION
**Pull request by iancw**
This allows specifying which network the chrome docker should connect too by including the `--net=${options.dockerNet}` flag in the docker runtime args. This allows loki to be used with tests that require communication with a running service, e.g. server side rendering.

### CODE REVIEW

File: packages/target-chrome-core/src/create-chrome-target.js [hunk 1]
```diff
@@ -202,22 +202,26 @@ function createChromeTarget(
...
         } catch (error) {
+          debug(`Error waiting for selector "${error.message}"!`);
           if (error.message.startsWith('Timeout')) {
             await ensureNoErrorPresent();
           }
           throw error;
         }
       }
 
+      debug(`Waiting for ensureNoErrorPresent...`);
```
Comment (techeverri) [inline packages/target-chrome-core/src/create-chrome-target.js:215]:
```suggestion
      debug('Waiting for ensureNoErrorPresent...');
```

File: packages/target-chrome-core/src/create-chrome-target.js [hunk 2]
```diff
@@ -202,22 +202,26 @@ function createChromeTarget(
...
       await ensureNoErrorPresent();
 
+      debug(`Waiting for awaitRequestsFinished...`);
```
Comment (techeverri) [inline packages/target-chrome-core/src/create-chrome-target.js:218]:
```suggestion
      debug('Waiting for awaitRequestsFinished...');
```

File: packages/target-chrome-core/src/create-chrome-target.js [hunk 3]
```diff
@@ -202,22 +202,26 @@ function createChromeTarget(
...
       await awaitRequestsFinished();
 
       debug('Awaiting runtime setup');
-
       await executeFunctionWithWindow(setLokiTestAttribute);
 
+      debug(`Waiting for executeFunctionWithWindow...`);
```
Comment (techeverri) [inline packages/target-chrome-core/src/create-chrome-target.js:224]:
```suggestion
      debug('Waiting for executeFunctionWithWindow...');
```

**Comment by techeverri** [comment]
The code looks go to me 👍 
_Thanks for updating the docs_ 💯 

I'm going to leave accepting the new parameter and merging to @oblador

**Review by oblador — APPROVED** [review]

**Comment by oblador** [comment]
Thanks!

**Pull request merged**